src/app/components/splash-screen/splash-screen.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'app-splash-screen',
+  templateUrl: './splash-screen.component.html',
+  styleUrls: ['./splash-screen.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SplashScreenComponent implements OnInit  {
+  windowWidth: string;
+  showSplash: boolean = true;
+
+  constructor(){}
+  ngOnInit(): void {
+    setTimeout (() => {
+      this.windowWidth = '-' + window.innerWidth + 'px';
+    },2000);
```

Comment:
If we need to use setTimeout(), please make sure to clean up the reference in OnDestroy hook.

Reply:
Why settimeout needed here?

Reply:
I have cleaned up using onDestroy hook Apu. 
It was needed for the splash screen animation vaia.